tests/desktop/test_switch_to_mobile_site.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from unittestzero import Assert
+from pages.desktop.base import Base
+from pages.desktop.page_provider import PageProvider
+from pages.mobile.page_provider import MobilePageProvider
+
+class TestMobileSite:
+
+    @pytest.mark.nondestructive
+    def test_switch_to_mobile_site(self, mozwebqa):
+        # Verify able to switch to mobile site using home page footer link
```

Comment:
Perhaps remove the unnecessary comment as the test is pretty much self-documenting.